gapic/utils/lines.py

```diff
@@ -86,8 +86,12 @@ def wrap(text: str, width: int, *, offset: Optional[int] = None, indent: int = 0
                                 break_on_hyphens=False,
                                 )
         # Strip the first \n from the text so it is not misidentified as an
-        # intentionally short line below.
-        text = text.replace('\n', ' ', 1)
+        # intentionally short line below, except when the text contains `:`
+        # as the new line is required for lists.
+        if '\n' in text:
+            initial_text = text.split('\n')[0]
+            if ":" not in initial_text:
```

Comment:
Could `:` appear on text line 2, and the list starts from line 3 and on?

Reply:
yes, I added a test case in e08978a